sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/complexTypeExtractors.scala

```diff
@@ -130,12 +134,17 @@ case class GetStructField(child: Expression, ordinal: Int, name: Option[String]
       }
     })
   }
+}
+
+case class PrettyGetStructField(child: Expression, name: String, dataType: DataType)
+  extends UnaryExpression with Unevaluable {
 
-  override def sql: String = child.sql + s".`${childSchema(ordinal).name}`"
+  override def sql: String = s"${child.sql}.$name"
```

Comment:
can we use this string as the name of `PrettyAttribute`? Then we can avoid creating these classes.